Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.14 as build-env
+WORKDIR /tmp/secrets-store-csi-driver-provider-gcp
+COPY . ./
+RUN CGO_ENABLED=0 go install github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp
```

Comment:
I think you're gonna want to make this fulllllllly static, enable go modules, strip it (and maybe upx it): https://github.com/google/exposure-notifications-server/blob/master/Dockerfile#L21-L35

Reply:
I'm not a fan of stripping the binary of debug symbols (especially during development) and don't want to take a hit on startup time since the binary would need to be unpacked for every mount operation.